playbooks/common/openshift-cluster/scaleup.yml

```diff
@@ -3,6 +3,4 @@
 
 - include: ../openshift-node/config.yml
   vars:
-    osn_cluster_dns_domain: "{{ hostvars[groups.oo_first_master.0].openshift.dns.domain }}"
-    osn_cluster_dns_ip: "{{ hostvars[groups.oo_first_master.0].openshift.dns.ip }}"
     openshift_deployment_type: "{{ deployment_type }}"
```

Comment:
These two show up in another file, should we remove them from there too?

https://github.com/openshift/openshift-ansible/blob/master/playbooks/gce/openshift-cluster/join_node.yml#L48-L49